module/support/optimisation/NSGA2Evaluator/src/NSGA2Evaluator.cpp

```diff
@@ -0,0 +1,316 @@
+#include "NSGA2Evaluator.hpp"
+
+#include <fmt/format.h>
+#include <fmt/ostream.h>
+#include <yaml-cpp/yaml.h>
+
+#include "tasks/RotationEvaluator.hpp"
+#include "tasks/StandEvaluator.hpp"
+#include "tasks/StrafeEvaluator.hpp"
+#include "tasks/WalkEvaluator.hpp"
+
+#include "extension/Configuration.hpp"
+#include "extension/Script.hpp"
+
+#include "message/behaviour/MotionCommand.hpp"
+#include "message/motion/WalkCommand.hpp"
+#include "message/platform/RawSensors.hpp"
+#include "message/platform/webots/WebotsResetDone.hpp"
+#include "message/platform/webots/WebotsTimeUpdate.hpp"
+#include "message/platform/webots/messages.hpp"
+#include "message/support/optimisation/NSGA2Evaluator.hpp"
+#include "message/support/optimisation/NSGA2Optimiser.hpp"
+
+#include "utility/behaviour/Action.hpp"
+#include "utility/behaviour/MotionCommand.hpp"
+#include "utility/input/LimbID.hpp"
+#include "utility/input/ServoID.hpp"
+#include "utility/support/yaml_expression.hpp"
+
+namespace module {
+    namespace support {
+        namespace optimisation {
+
+            using extension::Configuration;
+            using extension::ExecuteScriptByName;
+
+            using message::behaviour::MotionCommand;
+            using message::motion::DisableWalkEngineCommand;
+            using message::motion::EnableWalkEngineCommand;
+            using message::motion::StopCommand;
+            using message::motion::WalkCommand;
+            using RawSensorsMsg = message::platform::RawSensors;
```

Comment:
this looks unnecessary
```suggestion
            using message::platform::RawSensors;
```